middleware/proxy/proxy_test.go

```diff
@@ -1,148 +1,161 @@
 package proxy
 
-// // go test -run Test_Proxy_Empty_Host
-// func Test_Proxy_Empty_Host(t *testing.T) {
-// 	app := fiber.New()
-// 	app.Use(New(
-// 		Config{Hosts: ""},
-// 	))
-
-// 	resp, err := app.Test(httptest.NewRequest("GET", "/", nil))
-// 	utils.AssertEqual(t, nil, err)
-// 	utils.AssertEqual(t, fiber.StatusNotFound, resp.StatusCode)
-// }
+import (
+	"io/ioutil"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/recover"
+	"github.com/gofiber/fiber/v2/utils"
+)
+
+// go test -run Test_Proxy_Empty_Host
+func Test_Proxy_Empty_Upstream_Servers(t *testing.T) {
+	app := fiber.New()
+	app.Use(recover.New(), Balancer(Config{Servers: []string{}}))
+
+	resp, err := app.Test(httptest.NewRequest("GET", "/", nil))
+	utils.AssertEqual(t, nil, err)
+	utils.AssertEqual(t, fiber.StatusInternalServerError, resp.StatusCode)
+}
 
 // // go test -run Test_Proxy_Next
```

Comment:
Remove two forward slashes. Line 30 as well :)

Reply:
we should be good here, thanks